spec/gitpoints/points_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "spec_helper"
+
+describe GitPoints::UserPoints do
+  describe ".git_points" do
+    it "returns the points of the user" do
+      expect(git_points("test_user")).to eq 50
```

Comment:
@hrs Ah yea true about camel case not sure what i was doing (trying to make it like gitHub only gitHub is actually github... so )

Anyway...
it seams to be finding the file it was rasing an error on GitPoints, however its not running the test method?? 

```ruby
   expect(git_points("test_user")).to eq 50
```